### ISSUE TITLE
[Feature] a setup script

### ISSUE THREAD

**Issue by rotty3000** [furkilic/go-wrapper#10]
**Is your feature request related to a problem? Please describe.**
The problem is how to bootstrap the wrapper. It's not really too hard once you read a bit.

**Describe the solution you'd like**
However, It would be nice to have a one liner in the documentation, like a curl + shell/batch script, that automates adding the wrapper to a project (downloads the right files to the right places in a project).

**Comment by furkilic** [comment]
thanks again @rotty3000 

I let that here I will add it to the documentation tomorrow 

### Installation
- On Windows 
```bash
curl -LO https://raw.githubusercontent.com/furkilic/go-wrapper/master/gow.cmd
```
- Windows with GitBash or Cygwin, Linux, MacOs..
```bash
curl -LO https://raw.githubusercontent.com/furkilic/go-wrapper/master/gow
```

**Comment by rotty3000** [comment]
@furkilic I don't think it's really correct the way this is described.

The script isn't for individuals. It's for maintainers to setup their project to invite collaboration. A project maintainer doesn't choose one wrapper or the other. They setup the repository with wrappers for every operating system their community of developers might use.

So, the script should copy both wrapper scripts into the project, and it should copy the placeholder `./go/wapper/go-wrapper.properties`.

**Comment by furkilic** [comment]
![image](https://user-images.githubusercontent.com/1443467/192525353-3f13e2c2-c612-484c-aabc-021341a7128f.png)

@rotty3000  Yep you are absolutely right 
I was helping someone install kube client in parallel, got polluted with the idea.

**Comment by furkilic** [comment]
@rotty3000 I let you check #12
